manifests/init.pp

```diff
@@ -224,6 +226,7 @@
   Stdlib::IP::Address $http_host = '0.0.0.0',
   Stdlib::Port $http_port = 8080,
   Stdlib::Port $https_port = 8443,
+  String $http_relative_path = '/',
```

Comment:
```suggestion
  Stdlib::Unixpath $http_relative_path = '/',
```
While this isn't a Unix path, the fact this must start with "/" would have same validation behavior.